R-package/DESCRIPTION

```diff
@@ -7,9 +7,20 @@ Authors@R: c(
     person("Guolin", "Ke", email = "guolin.ke@microsoft.com", role = c("aut", "cre")),
     person("Damien", "Soukhavong", email = "damien.soukhavong@skema.edu", role = c("aut")),
     person("Yachen", "Yan", role = c("ctb")),
-    person("James", "Lamb", email="jaylamb20@gmail.com", role = c("aut"))
+    person("James", "Lamb", email="jaylamb20@gmail.com", role = c("aut")),
+    person("Qi", "Meng", role = c("aut")),
+    person("Thomas", "Finley", role = c("aut")),
+    person("Taifeng", "Wang", role = c("aut")),
+    person("Wei", "Chen", role = c("aut")),
+    person("Weidong", "Ma", role = c("aut")),
+    person("Qiwei", "Ye", role = c("aut")),
+    person("Tie-Yan", "Liu", role = c("aut")),
+    person("Microsoft Corporation", role = c("cph")),
+    person("Dropbox, Inc.", role = c("cph"))
```

Comment:
> we just have to be careful not to merge master into this branch any more so that the 3.0.0 release is really 3.0.0.

Just a note:
after merging this PR we will have to add IBM here
https://github.com/microsoft/LightGBM/blob/4278f22293e71ad7ff80f2133ff41abe4cced02b/src/treelearner/cuda_tree_learner.cpp#L1-L4

Reply:
oh right! Thank you for the note. I think we can actually do that as a separate PR right now into `master`. Since that copyright doesn't belong on this PR (with the 3.0.0 release).

Reply:
OK. If it doesn't cause any merging conflicts, I think we can do it right now then.

Reply:
I just added #3409 to do this. It will cause a merge conflict when I eventually merge `master` back into this branch, but that's not a problem. It will be a simple conflict to fix.